src/Components/FilterSidebar/CategoriesFilter.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+function CategoriesFilter() {
```

Comment:
I think we can use ES6 Syntax. But it's a small suggestion.

Reply:
Yes. I will change it